File: src/main/java/com/danidipp/sneakymisc/SneakyMiscBootstrap.java
package com.danidipp.sneakymisc;

import com.danidipp.sneakymisc.paintings.PaintingsBootstrapSupport;
import io.papermc.paper.plugin.bootstrap.BootstrapContext;
import io.papermc.paper.plugin.bootstrap.PluginBootstrap;

public final class SneakyMiscBootstrap implements PluginBootstrap {
    @Override
    public void bootstrap(final BootstrapContext context) {
        PaintingsBootstrapSupport.bootstrap(context);
    }
}

File: src/main/kotlin/com/danidipp/sneakymisc/SneakyMiscBootstrap.kt
package com.danidipp.sneakymisc

import com.danidipp.sneakymisc.paintings.PaintingsModule
import io.papermc.paper.plugin.bootstrap.BootstrapContext
import io.papermc.paper.plugin.bootstrap.PluginBootstrap

class SneakyMiscBootstrap : PluginBootstrap {
    override fun bootstrap(context: BootstrapContext) {
        PaintingsModule.bootstrap(context);
    }
}